enclave_manager/Dockerfile

```diff
@@ -171,19 +171,21 @@ RUN mkdir -p /opt/intel \
  && mkdir -p /etc/init
 WORKDIR /opt/intel
 
-# Install Intel SGX common library
-RUN wget https://download.01.org/intel-sgx/sgx-linux/2.7.1/distro/ubuntu18.04-server/libsgx-enclave-common_2.7.101.3-bionic1_amd64.deb \
- && dpkg -i libsgx-enclave-common_2.7.101.3-bionic1_amd64.deb
+RUN wget https://download.01.org/intel-sgx/sgx-linux/2.10/distro/ubuntu18.04-server/debian_pkgs/libs/libsgx-enclave-common/libsgx-enclave-common_2.10.100.2-bionic1_amd64.deb \
+ && wget https://download.01.org/intel-sgx/latest/linux-latest/distro/ubuntu18.04-server/debian_pkgs/libs/libsgx-epid/libsgx-epid_2.10.100.2-bionic1_amd64.deb \
+ && wget https://download.01.org/intel-sgx/latest/linux-latest/distro/ubuntu18.04-server/debian_pkgs/libs/libsgx-launch/libsgx-launch_2.10.100.2-bionic1_amd64.deb \
+ && wget https://download.01.org/intel-sgx/latest/linux-latest/distro/ubuntu18.04-server/debian_pkgs/libs/libsgx-urts/libsgx-urts_2.10.100.2-bionic1_amd64.deb \
+ && dpkg -i *
```

Comment:
The installation approach used in Dockerfile is different from the one mentioned in PREREQUISITES.md (which uses  `sudo apt-get install libsgx-launch libsgx-epid libsgx-urts`). Though there could be multiple options can we follow one consistent approach

Reply:
apt based installation takes care of all necessary dependent sgx debians implicitly. Hence apt based approach is followed to ease downloading and installing of multiple packages. However within docker only few debian packages are needed, hence i used different approach. But i agree, it makes sense to use consistent approach for better clarity. I will rectify it.